src/AbstractPolytope.jl

```diff
@@ -81,3 +84,25 @@ function linear_map(M::AbstractMatrix, P::AbstractPolytope{N}) where {N<:Real}
     end
     return T(new_vlist)
 end
+
+"""
+    isempty(P::AbstractPolytope{N})::Bool where {N<:Real}
+
+Determine whether a polytope is empty.
+
+### Input
+
+- `P` -- abstract polytope
+
+### Output
+
+`true` if the given polytope contains no vertices, and `false` otherwise.
+
+### Algorithm
+
+This algorithm checks whether the `vertices_list` of the given polytope is empty
+or not.
+"""
+function isempty(P::AbstractPolytope{N})::Bool where {N<:Real}
+    return isempty(vertices_list(P))
+end
```

Comment:
I agree that the `HPolytope` with no constraints is particular case that could be treated separately as being "universal", the whole R^n. 

For `VPolytope`, i think it is reasonable to say that if it has no vertices then it only contains the empty set, `V = convex_hull{{}}`.

I was concerned with the case where the set is defined by a set of constraints with empty intersection. For instance consider the pair of constraints `x <= -1 && x >= 1`:

```julia
julia> P = HPolytope([LazySets.HalfSpace([1.0, 0.0], -1.0), LazySets.HalfSpace([-1.0, 0.0], -1.0)])
LazySets.HPolytope{Float64}(LazySets.HalfSpace{Float64}[LazySets.HalfSpace{Float64}([1.0, 0.0], -1.0), LazySets.HalfSpace{Float64}([-1.0, 0.0], -1.0)])

julia> isempty(P)
true

julia> vertices_list(P)
0-element Array{Array{Float64,1},1}
```

Reply:
Rather than finding workarounds, we could accept that our algorithms are generally not correct for invalid polytope inputs, like unbounded or contradicting constraints :wink:

> For VPolytope, i think it is reasonable to say that if it has no vertices then it only contains the empty set, V = convex_hull{{}}.

Sure.

If `HPolytope` and `HPolygon` are problematic, should we replace `AbstractPolytope` by `Union{VPolygon, VPolytope}` then? Our other polytopic sets can never be empty by definition anyway.